flexmeasures/cli/data_show.py

```diff
@@ -234,7 +234,12 @@ def list_data_sources(source: DataSource | None = None, show_attributes: bool =
     Show available data sources
     """
     if source is None:
-        sources = DataSource.query.order_by(DataSource.name).all()
+        sources = (
+            DataSource.query.order_by(DataSource.name)
```

Comment:
I think if you `order_by(DataSource.type)` first (and then these others), your table headers should become alphabetically sorted, too.

Reply:
Good idea